examples/pytorch/PSS/README.md

```diff
@@ -0,0 +1,62 @@
+# PSS
+
+Code for the ECCV '22 submission "PSS: Progressive Sample Selection for Open-World Visual Representation Learning".
+
+## Dependencies
+
+We use python 3.7. The CUDA version needs to be 10.2. Besides DGL (>=0.5.2), we depend on several packages. To install dependencies using conda:
```

Comment:
Can you be more precise about DGL version? Recently there's a DGL upgrade that changes dataloader interface.